src/burn/drv/irem/d_m92.cpp

```diff
@@ -9,7 +9,7 @@
 #include "msm6295.h" // ppan
 #include "irem_cpu.h"
 #include "iremga20.h"
-#include "stddef.h"
+#include <cstddef>
 
```

Comment:
no, please don't change that - it's not compatible with super ancient compilers that some systems need to use :)